extensions/hibernate-search-orm-elasticsearch/deployment/pom.xml

```diff
@@ -52,6 +67,118 @@
                     </annotationProcessorPaths>
                 </configuration>
             </plugin>
+            <plugin>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <configuration>
+                    <skip>true</skip>
```

Comment:
Is this intended?

Reply:
Yes, they need Elasticsearch so they are run if `-Dtest-containers`